src/rust-crypto/rust-crypto.ts

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as RustSdkCryptoJs from "@matrix-org/matrix-sdk-crypto-js";
+
+import { IEventDecryptionResult } from "../@types/crypto";
+import { MatrixEvent } from "../models/event";
+import { CryptoBackend } from "../common-crypto/CryptoBackend";
+
+// import { logger } from "../logger";
+
+/**
+ * An implementation of {@link CryptoBackend} using the Rust matrix-sdk-crypto.
```

Comment:
```suggestion
 * An implementation of {@link CryptoBackend} using the Rust matrix-sdk-crypto.
 * @experimental This entire class is subject to breaking changes, without notice.
```

or whatever jsdoc wants to say this.

Reply:
alternatively, apparently `@internal` will exclude it from public types, despite being public itself.

Reply:
Well, it's not exported explicitly, and it's only dynamically referenced, so it doesn't actually appear in the generated documentation.